iac-extensions/terraform/src/test/resources/checks/GCP/ComputeInstanceSshKeysCheck/ssh_keys.tf

```diff
@@ -0,0 +1,63 @@
+resource "google_compute_instance" "example1" { # Noncompliant {{Omitting metadata.block-project-ssh-keys enables project-wide SSH keys. Make sure it is safe here.}}
+#        ^^^^^^^^^^^^^^^^^^^^^^^^^
+}
+
+resource "google_compute_instance" "example2" {
+  metadata = true # Noncompliant {{metadata of type Object expected.}}
+# ^^^^^^^^^^^^^^^
+}
+
+resource "google_compute_instance" "example3" {
+  metadata = { ssh-keys = true } # Noncompliant {{Omitting metadata.block-project-ssh-keys enables project-wide SSH keys. Make sure it is safe here.}}
+# ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
```

Comment:
I would not report the entire object. Maybe keep it the same way we raise items in lists or attributes in blocks.